charts/localstack/Chart.yaml

```diff
@@ -18,5 +18,5 @@ maintainers:
   email: waldemar.hummer@gmail.com
```

Comment:
Need to rev the chart version to `0.4.0` probably (since you're adding a new feature)